pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BacklogQuotaManager.java

```diff
@@ -103,6 +105,7 @@ public void handleExceededBacklogQuota(PersistentTopic persistentTopic, BacklogQ
             break;
         case producer_exception:
         case producer_request_hold:
+            advanceSlowestMessageDeduplicationCursor(persistentTopic, backlogQuotaType);
```

Comment:
```suggestion
            advanceSlowestSystemCursor(persistentTopic, backlogQuotaType);
```
Because we also need to fix other system cursors, such as topic compaction.